app/models/find_api/site.rb

```diff
@@ -2,5 +2,15 @@ module FindAPI
   class Site < FindAPI::Resource
     belongs_to :recruitment_cycle, through: :provider, param: :recruitment_cycle_year
     belongs_to :provider, param: :provider_code
+
+    def name
+      location_name
+    end
+
+    def full_address
+      [address1, address2, address3, address4, postcode]
+        .reject(&:blank?)
+        .join(', ')
+    end
```

Comment:
Hey, we're moving away from using the v3 API. Can we use this instead please:

![image](https://user-images.githubusercontent.com/42515961/107526070-5ede2e00-6baf-11eb-863d-c9c13186b38f.png)